.travis.yml

```
@@ -113,13 +113,15 @@ matrix:
         - source_pkg gcc
         - arm-none-eabi-gcc --version
         - pip install --user --upgrade cmake
+        - pip install --user --upgrade mbed-tools
         - >-
          git clone --depth=1 --single-branch --branch feature-cmake https://github.com/ARMmbed/mbed-os-example-blinky.git;
          cd mbed-os-example-blinky;
          git clone --depth=1 --single-branch --branch feature-cmake https://github.com/ARMmbed/mbed-os.git;
+         mbedtools configure -t GCC_ARM -m K64F;
          mkdir -p build;
       script:
-        - cd build && cmake .. -GNinja && ninja
+        - cd build && cmake .. -GNinja -DMBED_PROFILE=develop && ninja
```

```suggestion
        - cd build && cmake .. -GNinja -DMBED_PROFILE=develop && cmake --build .
```
This command is a generic, cross-platform wrapper to the native build command for the chosen generator, `ninja` in this case.

---

updated